apps/web/components/connected/Landspitali/MemorialCard.tsx

```diff
@@ -0,0 +1,363 @@
+            {formatMessage(m.overview.goBack)}
+          </Button>
+          <Button>{formatMessage(m.overview.pay)}</Button>
+        </Box>
+      </Stack>
+    )
+  }
+
+  const requiredRule = {
+    required: { value: true, message: 'Það þarf að fylla út þennan reit' },
+  }
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Stack space={4}>
+          <SelectController
+            id="fund"
+            name="fund"
+            label={formatMessage(m.info.fundLabel)}
+            options={fundOptions}
+            defaultValue={fundOptions.find(
+              (opt: { value: string }) => opt.value === watch('fund'),
+            )}
+            onSelect={(opt) => setValue('fund', (opt?.value as string) || '')}
+            error={errors.fund?.message}
+            rules={requiredRule}
+          />
+
+          <InputController
+            id="inMemoryOf"
+            control={control}
+            label={formatMessage(m.info.inMemoryOfLabel)}
+            size="xs"
+            error={errors.inMemoryOf?.message}
+            rules={requiredRule}
+          />
+
+          <Text variant="h2">{formatMessage(m.info.amountOfMoneyTitle)}</Text>
+          <Text>{formatMessage(m.info.amountOfMoneyExtraInfo)}</Text>
+
+          <Box display="flex" flexWrap="wrap" columnGap={2} rowGap={2}>
+            {PRESET_AMOUNTS.map((amount) => (
+              <FocusableBox
+                key={amount}
+                border="standard"
+                borderRadius="standard"
+                padding={2}
+              >
+                <RadioButton
+                  id={`amount-${amount}`}
+                  name="amountISK"
+                  label={parseInt(amount)}
+                  checked={selectedAmount === amount}
+                  onChange={() => setValue('amountISK', amount)}
+                />
+              </FocusableBox>
+            ))}
+            <FocusableBox border="standard" borderRadius="standard" padding={2}>
+              <RadioButton
+                id="amount-other-radio"
+                name="amountISK"
+                label={formatMessage(m.info.amountOfMoneyOtherRadioLabel)}
+                checked={!PRESET_AMOUNTS.includes(selectedAmount)}
+                onChange={() => setValue('amountISK', '')}
+              />
+            </FocusableBox>
+          </Box>
+
+          {!PRESET_AMOUNTS.includes(selectedAmount) && (
+            <InputController
+              id="amountISK"
+              control={control}
+              label={formatMessage(m.info.amountOfMoneyOtherInputLabel)}
+              type="number"
+              inputMode="numeric"
+              currency={true}
+              error={
+                parseInt(selectedAmount) < 1000
+                  ? 'Lágmark er 1.000 kr.'
+                  : undefined
+              }
+              rules={requiredRule}
+            />
+          )}
+
+          <InputController
```

Comment:
I think it'd be better if we move this field right below "Til minningar um" to not confuse the user as much when a new input field appears if he wants to type in another amount.